editor/scene_tree_dock.h

```diff
@@ -152,9 +152,16 @@ class SceneTreeDock : public VBoxContainer {
 	ConfirmationDialog *delete_dialog = nullptr;
 	Label *delete_dialog_label = nullptr;
 	CheckBox *delete_tracks_checkbox = nullptr;
+	ConfirmationDialog *editable_instance_enable_or_disable_dialog = nullptr;
+	Button *editable_instance_disable_all_button = nullptr;
 	ConfirmationDialog *editable_instance_remove_dialog = nullptr;
+	ConfirmationDialog *placeholder_enable_or_disable_dialog = nullptr;
+	Button *placeholder_disable_all_button = nullptr;
 	ConfirmationDialog *placeholder_editable_instance_remove_dialog = nullptr;
 
+	bool editable_instance_change_operation_is_enable = false;
+	bool placeholder_change_operation_is_enable = false;
```

Comment:
```suggestion
	bool editable_instance_change_operation_is_enabled = false;
	bool placeholder_change_operation_is_enabled = false;
```

Reply:
These I use to temporarily store the result of one dialog if there might be another dialog before doing the operation, so the meaning is less "a something is enabled" and more "the thing we are about to do is enable". 

The suggested change to "enabled" might imply that nothing will happen if it is false (e.g. "the operation is not enabled"), when it instead means that the user chose to disable all rather than enable all.

How about `editable_instance_enable_or_disable_dialog_result_was_enable` and `placeholder_enable_or_disable_dialog_result_was_enable` ? Bit of a mouthful, but purpose more explicit.

Or, remove the `_was_enable` and use an enum `{DISABLE_ALL, ENABLE_ALL}` instead?

Actually, since only one of these will be in play at any given time, could just roll into a single `dialog_result_was_enable` bool?

Reply:
Yep, I think they should be unified. Maybe something like `dialog_option_enable_all`? Though "dialog" is a bit ambiguous, but I don't have better idea (that isn't super long).